packages/itwin/ec3-widget/src/components/LabelActionPanel.tsx

```diff
@@ -0,0 +1,59 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Button } from "@itwin/itwinui-react";
+import * as React from "react";
+import "./LabelActionPanel.scss";
+import { LoadingSpinner } from "./utils";
+import {
+  SvgExit,
+  SvgSave,
+} from "@itwin/itwinui-icons-react";
+
+export interface ActionPanelProps {
+  onSave: () => void;
+  onCancel: () => void;
+  isCancelDisabled?: boolean;
+  isSavingDisabled?: boolean;
+  isLoading?: boolean;
+}
+
+const ActionPanel = ({
+  onSave,
+  onCancel,
+  isCancelDisabled = false,
+  isSavingDisabled,
+  isLoading = false,
+}: ActionPanelProps): JSX.Element => {
+  return (
+    <div id='action'>
```

Comment:
I think this `id` is not used anywhere. Remove it.

Reply:
Is this whole `div` even needed?

Reply:
removed the div